src/components/StyleGuide/StyleGuide.scss

```diff
@@ -0,0 +1,101 @@
+@import '~style/variables';
+
+.pageWrapper {
+  font-size: 5px; // to ensure font-sizes are set on each mixin
+  padding: 25px 25px 200px;
+  background: $color-white;
+}
+
+.colorsWrapper {
+  margin-bottom: 100px;
+}
+
+.sectionText {
+  @include type-h1;
+}
+
+.subSectionText {
+  @include type-h3;
+}
+
```

Comment:
Again, how about DRYing this up? @each loop over a $colors array?

Reply:
I gave it 15 minutes of trying then gave up. Using interpolation to get the variable names (in the `@each`) doesn't seem possible.